### ISSUE TITLE
Epub?

### ISSUE THREAD

**Issue by IAmAngelN** [fnzr/minha-biblioteca-jigsaw#4]
olá, alguma idéia se isso vai funcionar se o arquivo é epub e não pdf? Também desculpe se o português está errado, estou usando o google tradutor

**Comment by fnzr** [comment]
Deve funcionar, mas não fiz muitos testes com epub. Me passa o título do livro que eu posso testar aqui.
E pode responder em inglês se for mais fácil para você.

**Comment by IAmAngelN** [comment]
thank you for the reply! Its a textbook i need for class its title is The Curious Writer, Brief edition its isbn is 9780134120744

**Comment by fnzr** [comment]
Can you access this book through the browser? I'm getting a "Sorry, you don't have access to that book" error message, which is not related to the script itself, but a server side restriction.

**Comment by IAmAngelN** [comment]
Yes I can access the book, I can send you my credentials. I have it in my library which is why you’ll probably need me to be signed in. Thanks again!

**Comment by amlemos** [comment]
Por favor alguém pode me ajudar, fiz todo o processo, chega a criar a pasta com o ano, mês dia e tempo, porém tudo vazio.
